paper.md

```diff
@@ -35,65 +35,75 @@ bibliography: paper.bib
+DataLad aims to solve these issues by streamlining data consumption, publication, and updating routines, by providing a simplified core API for Git and git-annex operations, and by providing additional features for reproducible science such as command execution <!-- Maybe "execution provenance capture", datalad does not bring "command execution" --> and automatic recomputation based on complete process provenance.
+As it is interoperable with a large variety of scientific and commercial hosting services and available for all operating systems.
+DataLad can be integrated into established systems with minimal adjustments.
+
+## Why Git and git-annex
+
+Git is an excellent <!-- citation needed -->distributed content management system geared towards managing and collaborating on text files <!-- citation needed, does not match self-description -->, but as such it cannot handle large <!-- what is large? --> or binary files well <!-- how come? I have hundreds of repos with binary files in them, works just fine -->.
+Moreover, any data committed to Git becomes available to all clones of that repository. In the context of scientific data, this makes it hard or impossible to provide distributed storage of managed data <!-- why? individual tailoring is hard, but distribution not, what is special about scientific data in this regard? -->, or revoke individual files, such as accidentally stored personal data or data from participants that withdrew from a study.
+Git-annex takes advantage of Git's ability to manage textual information to overcome Git's limitation in managing individual file content that is either large or sensitive.
+File content managed by git-annex is placed into an annex, and in its place, git-annex commits a symbolic link <!-- platform-dependent implementation details, "pointer" could be a term to illustrate the concept and add the "symbolic link" as an example implmentation --> pointing to the content of the file into Git.
 The content representation is based on a checksum of the file's content.
-As such, while only a light-weight link is directly committed into git, this committed information contains the content identity of annexed files.
-Git-annex then manages files availability information in any local repository, other git remotes, or external resources, e.g. web urls.
-Such simple approach allows git-annex to scale to manage virtually arbitrarily large files, and "link" files in a git repository to vast data resources available online. 
+As such, while only a light-weight link <!-- pointer --> is directly committed into Git, this committed information contains the content identity of annexed files.
+Git-annex then manages files availability information in any local repository, other Git remotes, or external resources, e.g. web urls. <!-- git annex also takes care of transport logistics and provides the abstraction and implementations of a special remote -->
```

Comment:
```suggestion
Git-annex then manages files availability information in any local repository, other Git remotes, or external resources, e.g. web urls.
Having that information, git-annex takes care of all transport logistics to exchange data upon user request.
```

re `special remote` -- might derail a bit with technical detail and we do mention special remotes later